src/tests/commonTests.ts

```diff
@@ -306,64 +306,74 @@ export function focusable(componentTagOrHTML: TagOrHTML, options?: FocusableOpti
 
-  await page.$eval(
-    tag,
-    async (component, slotNames: string[], includeDefaultSlot?: boolean) => {
-      async function slotTestElement(testClass: string, slotName?: string): Promise<void> {
-        const el = document.createElement("div"); // slotting a <div> will suffice for our purposes
-        el.classList.add(testClass);
+    await page.$eval(
+      tag,
+      async (component, slotNames: string[], includeDefaultSlot?: boolean) => {
+        async function slotTestElement(testClass: string, slotName?: string): Promise<void> {
+          const el = document.createElement("div"); // slotting a <div> will suffice for our purposes
+          el.classList.add(testClass);
 
-        if (slotName) {
-          el.slot = slotName;
-        }
+          if (slotName) {
+            el.slot = slotName;
+          }
 
-        component.append(el);
-        await new Promise<void>((resolve) => requestAnimationFrame(() => resolve()));
-      }
+          component.append(el);
+          await new Promise<void>((resolve) => requestAnimationFrame(() => resolve()));
+        }
 
-      for (let i = 0; i < slotNames.length; i++) {
-        await slotTestElement("slotted-into-named-slot", slotNames[i]);
-      }
+        for (let i = 0; i < slotNames.length; i++) {
+          await slotTestElement("slotted-into-named-slot", slotNames[i]);
+        }
 
-      if (includeDefaultSlot) {
-        await slotTestElement("slotted-into-default-slot");
-      }
-    },
-    slotNames,
-    includeDefaultSlot
-  );
+        if (includeDefaultSlot) {
+          await slotTestElement("slotted-into-default-slot");
+        }
+      },
+      slotNames,
+      includeDefaultSlot
+    );
 
-  await page.waitForChanges();
+    await page.waitForChanges();
 
-  const slotted = await page.evaluate(() =>
-    Array.from(document.querySelectorAll(".slotted-into-named-slot"))
-      .filter((slotted) => slotted.assignedSlot)
-      .map((slotted) => slotted.slot)
-  );
+    const slotted = await page.evaluate(() =>
+      Array.from(document.querySelectorAll(".slotted-into-named-slot"))
+        .filter((slotted) => slotted.assignedSlot)
+        .map((slotted) => slotted.slot)
+    );
 
-  expect(slotNames).toEqual(slotted);
+    expect(slotNames).toEqual(slotted);
 
-  if (includeDefaultSlot) {
-    const hasDefaultSlotted = await page.evaluate(() => {
-      const defaultSlotted = document.querySelector(".slotted-into-default-slot");
-      return defaultSlotted.assignedSlot?.name === "" && defaultSlotted.slot === "";
-    });
+    if (includeDefaultSlot) {
+      const hasDefaultSlotted = await page.evaluate(() => {
+        const defaultSlotted = document.querySelector(".slotted-into-default-slot");
+        return defaultSlotted.assignedSlot?.name === "" && defaultSlotted.slot === "";
+      });
 
-    expect(hasDefaultSlotted).toBe(true);
-  }
+      // eslint-disable-next-line jest/no-conditional-expect
```

Comment:
Can you add an explainer to this? See https://eslint.org/docs/latest/use/configure/rules#comment-descriptions.